### DISCUSSION TITLE
Control how date display works across app (createdAt, updatedAt) in admin panel

### DISCUSSION THREAD

**Discussion by dio-ttdib** [payloadcms/payload#6142] [Q&A]
Hello everyone,
I would like to ask if it possible to configure date display format in our admin panel (createdAt, updatedAt, versions)? 
i can see the default format is MM-DD-YYYY.
How do i apply global format of DD-MM-YYYY H:M a to all of my payload app? 

Of course we can specify date format from our date field, but i think the use case is much more than that.

hope everyone can help with this, thankyou 🙏

**Comment by tsemachh** [comment]
I would suggest using moment.js to give meaningful dates(Or at least enable it)
here's what we did in our case:
```typescript
export const CreatedAtCell: React.FC = () => {
  const { cellData } = useTableCell()
  // Ensure cellData is not null before passing it to moment
  const validCellData = cellData !== null ? cellData : undefined

  return <Fragment>{validCellData ? moment(validCellData).fromNow() : 'N/A'}</Fragment>
}
```

**Reply by tsemachh** [reply]
@paulpopus I think it's a small change and would give payload professional look

**Reply by andershermansen** [reply]
You already have this. Put a date-fns format in your payload.config.

```
    admin: {
        dateFormat: "d. MMMM y 'kl.' HH:mm",
    },
```

Documentation at: https://payloadcms.com/docs/admin/overview#admin-options

**Reply by tsemachh** [reply]
Not as good as moment.js you get:
![image](https://github.com/user-attachments/assets/fd73bc1e-6cc4-4679-91a7-f7df4bcd38fb)
